specification/monitor/resource-manager/readme.go.md

```diff
@@ -130,4 +136,58 @@ Please also specify `--go-sdk-folder=<path to the root directory of your azure-s
 
 ``` yaml $(tag) == 'package-2021-09' && $(go)
 output-folder: $(go-sdk-folder)/services/preview/monitor/mgmt/2021-09-01-preview/$(namespace)
-```
\ No newline at end of file
+```
+
+### Tag: package-2022-03-preview-datacollection and go
+
+These settings apply only when `--tag=package-2022-03-preview-datacollection --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2022-03-preview-datacollection' && $(go)
+output-folder: $(go-sdk-folder)/services/preview/monitor/mgmt/2022-03-26-preview/$(namespace)
+```
+
+### Tag: package-2022-03-part5 and go
+
+These settings apply only when `--tag=package-2022-03-part5 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2022-03-part5' && $(go)
+output-folder: $(go-sdk-folder)/services/monitor/mgmt/2022-03-25/$(namespace)
```

Comment:
Since this tag only contains one file
Microsoft.Insights/stable/2021-08-01/scheduledQueryRule_API.json
how about 
```suggestion
output-folder: $(go-sdk-folder)/services/monitor/mgmt/2021-08-01/scheduledqueryrules
```